libs/x-content/src/main/java/org/elasticsearch/common/xcontent/XContentBuilder.java

```diff
@@ -550,6 +563,10 @@ public XContentBuilder value(short value) throws IOException {
     // String
     //////////////////////////////////
 
+    public XContentBuilder field(ParseField field, String value) throws IOException {
+        return field(field.getPreferredName(), value);
+    }
+
```

Comment:
nit: I wonder what these method buy us, my concern is that we have already a lot of methods in XContentBuilder. After all callers only have to call the other variant of these methods and pass in `field.getPreferredName()`. I am a bit on the fence on this. Maybe let's try and do it separately and see what others think about it.